bindings/capacitorjs/src/definitions.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { EventType, AccountManagerOptions } from './types';
+export interface IotaWalletMobileTypes {
+    initLogger(path: string): Promise<void>;
+    messageHandlerNew(messageOptions: AccountManagerOptions): Promise<{
+        messageHandler: number;
+    }>;
+    sendMessage(messageOptions: {
+        message: string;
+        handler: number;
+    }): Promise<{
+        result: string;
+    }>;
+    listen(options: {
+        eventTypes: EventType[];
+        messageHandler: number;
+    }, callback: (message: {
+        error: {
+            cause: unknown;
+        };
+        result: string;
+    }) => void): Promise<void>;
+    clearListeners(options: {
+        eventTypes: EventType[];
+        messageHandler: number;
```

Comment:
I don't think you need `messageHandler` here. 

https://github.com/amadeu2/wallet.rs/blob/33740c11f28aa8f68bc31776bc786e5795f468fe/src/account_manager/mod.rs#L224

Reply:
It's needed because of this
https://github.com/amadeu2/wallet.rs/blob/6cc138fc4c4e05c0624c149217fb3e3dc062f5fb/bindings/swift/src/lib.rs#L218
Now the `iota_clear_listeners` was removed in favor to use it with `iota_send_message`, but consider that the Android and iOS built libs which work with this version include it

Reply:
with latest updates `clearLiseners` was removed